reference-architecture/vmware-ansible/roles/openshift-default-selector/tasks/main.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Set the default node selector
+  replace:
+     dest: /etc/origin/master/master-config.yaml
+     regexp: 'defaultNodeSelector: ""'
+     replace: 'defaultNodeSelector: "role=app"'
```

Comment:
they just need to be set on the hosts (or otherwise available) during the invocation of the byo config playbook.